src/services/__tests__/tourettes.test.js

```diff
@@ -18,31 +18,49 @@ describe('tourettes', () => {
   describe('sayTourettes', () => {
     describe('when input text is empty', () => {
       const inputText = '';
+      const minAmountOfReplies = 3;
 
       it('replies with at least three items from the profanities list', () => {
         const reply = sayTourettes(inputText);
+        const amountOfReplies = reply.split(' ').length;
         expect(reply).toNotBe(undefined);
-        expect(reply.split(' ').length).toBeGreaterThan(3);
+        expect(amountOfReplies).toBeGreaterThanOrEqualTo(minAmountOfReplies);
+      });
+    });
+
+    describe('when input text is integer', () => {
+      const amountOfProfanities = 25;
```

Comment:
I'm a bit sceptic to put a specific number in tests like this, it raises questions like "why 25? does 25 have a special meaning?" etc.

I usually like to keep it as simple as possible, if I have to specify a specific number I try to go with 1 for instance, just because it's the first number, the number itself isn't really important, only that it is a number.

No need to change if you disagree, just wanted to float my opinion :)